Edge AI Readme improvements (#1953)
microsoft/azure_arc: docs/azure_jumpstart_ag/contoso_supermarket/ai/_index.md

One of the main benefits of GitOps is its ability to provide a consistent and auditable deployment process. All changes to the infrastructure and application configurations are tracked and versioned in Git, making it easy to roll back to a previous state if necessary. The declarative nature of GitOps also enables the automation of the deployment process, which reduces the risk of human error and streamlines the entire process.

> ## AI at the Edge using Intel OpenVino

Reviewer: ```suggestion
### AI at the Edge using Intel OpenVino
```

> Contoso aims to maximize the usage of their existing CCTV cameras installed across their stores. After careful evaluation, Contoso has opted to utilize Intel's OpenVino Technology for their solution. By leveraging OpenVino, Contoso can perform inference on the CPU of their edge devices, eliminating the need for expensive discrete GPUs. Intel also provides a set of pretrained models that align with Contoso's requirements.

Reviewer: ```suggestion
Contoso aims to maximize the usage of their existing Closed-circuit television (CCTV) cameras installed across their stores. After careful evaluation, Contoso has opted to utilize Intel's OpenVino Technology for their solution. By leveraging OpenVino, Contoso can perform inference on the CPU of their edge devices, eliminating the need for expensive discrete GPUs. Intel also provides a set of pre-trained models that align with Contoso's requirements.
```

    ![Screenshot showing the Store Manager Screenshot Heatmap focus](./img/checkout-heatmap.png)

- __Reporting:__ Contoso is interested in improving their store experience by measuring two key metrics: Total people in the store, and Shopper Wait Time. These metrics are shown in the Store Manager Control Center as shown below.

    ![Screenshot showing the Store Manager Reporting Pane](./img/reporting.png)

## Live View

These metrics are generated by combining an AI engine that captures inferences from the CCTV cameras with an application running the business logic. To provide visibility into the process of capturing these inferences, the Contoso Dev team has included an option to view the live video feed from one of their cameras.

Contoso has decided to have the "Live View" option enabled by default in the Dev and Staging environments.

![Screenshot showing the enabled live view button](./img/live-view-button.png)

Live View will be disabled by default in Seattle and Chicago environments as shown in the picture below:

![Screenshot of disabled live view button](./img/disabled-live-view.png)

### GitOps workflow

> Contoso has deployed GitOps configurations for all of their services to make it easier for their developers to focus on code and streamline all the code build, test, and deployment activities. Flux v2 is the main driver for Contoso's GitOps workflow by monitoring the Git repository holding all the clusters' configurations and applications code for changes and automatically updating the deployed resources accordingly. This makes it easy for teams to manage their infrastructure as code, using Git as the single source of truth. [GitOps on Azure Arc-enabled Kubernetes](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-kubernetes/eslz-arc-kubernetes-cicd-gitops-disciplines) leverages Flux v2 to provide a unified experience for managing Kubernetes clusters across different environments. This integration allows users to deploy and manage applications in Arc-enabled Kubernetes clusters, while also providing a centralized management plane through Azure Arc. This can simplify the process of managing Kubernetes clusters across different environments and provide greater visibility and control over the entire deployment process.

Reviewer: ```suggestion
Contoso has deployed GitOps configurations for all of their services to make it easier for their developers to focus on code and streamline all the code build, test, and deployment activities. Flux v2 is the main driver for Contoso's GitOps workflow by monitoring the Git repository holding all the clusters' configurations and applications code for changes and automatically updating the deployed resources accordingly. This makes it easy for teams to manage their infrastructure as code, using Git as the single source of truth. 

[GitOps on Azure Arc-enabled Kubernetes](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/hybrid/arc-enabled-kubernetes/eslz-arc-kubernetes-cicd-gitops-disciplines) leverages Flux v2 to provide a unified experience for managing Kubernetes clusters across different environments. This integration allows users to deploy and manage applications in Arc-enabled Kubernetes clusters, while also providing a centralized management plane through Azure Arc. This can simplify the process of managing Kubernetes clusters across different environments and provide greater visibility and control over the entire deployment process.
```
